cluster/node-pools/master-default/userdata.yaml

```diff
@@ -211,7 +211,7 @@ write_files:
             requests:
               cpu: 100m
               memory: 200Mi
-        - image: registry.opensource.zalan.do/teapot/admission-controller:master-94
+        - image: registry.opensource.zalan.do/teapot/admission-controller:master-93
```

Comment:
I've meant here, this will avoid an unnecessary update of the masters.

Reply:
Fixed